src/cmd/alias.go

```diff
@@ -1,8 +1,8 @@
 package cmd
 
 import (
-	"github.com/Originate/git-town/lib/git"
-	"github.com/Originate/git-town/lib/script"
+	"github.com/Originate/git-town/src/git"
```

Comment:
The default setup for Cobra is to have a `cmd` folder in the root directory. That's at least what their code generator creates. Is there a need to move everything to `src`? I'm not against it, just curious why we break the convention here.